lib/src/platform.ts

```diff
@@ -0,0 +1,20 @@
+// export client service definitions
+export * from './platform/authorization/authorization_pb.js';
+export * from './platform/common/common_pb.js';
+export * from './platform/entityresolution/entity_resolution_pb.js';
+export * from './platform/kas/kas_pb.js';
+export * from './platform/policy/actions/actions_pb.js';
+export * from './platform/policy/attributes/attributes_pb.js';
+export * from './platform/policy/kasregistry/key_access_server_registry_pb.js';
+export * from './platform/policy/namespaces/namespaces_pb.js';
+export * from './platform/policy/objects_pb.js';
+export * from './platform/policy/registeredresources/registered_resources_pb.js';
+export * from './platform/policy/resourcemapping/resource_mapping_pb.js';
+export * from './platform/policy/selectors_pb.js';
+export * from './platform/policy/subjectmapping/subject_mapping_pb.js';
+export * from './platform/policy/unsafe/unsafe_pb.js';
+export * from './platform/wellknownconfiguration/wellknown_configuration_pb.js';
+
+// export Connect RPC framework
+export * from '@connectrpc/connect-web';
+export * from '@connectrpc/connect';
```

Comment:
Shouldn't these be export as? It seems like this will result in a lot of overly simplified names.

Reply:
> Shouldn't these be export as?
I've had that thought too, currently there is no name collision but there might be one eventually.  I'll update to use a "name" export. As a result,  the imports will look like 
```
import { wellknown.WellKnownService }from "@opentdf/sdk/platform"
```
```
export * wellknown from './platform/wellknownconfiguration/wellknown_configuration_pb.js';
```